server/routes/main/refresh.js

```diff
@@ -0,0 +1,26 @@
+var querystring = require("querystring");
+
+module.exports = function(config, req, res) {
+  var locale = (req.localeInfo && req.localeInfo.lang) ? req.localeInfo.lang : "en-US";
+  var qs = querystring.stringify(req.query);
+  if(qs !== "") {
+    qs = "?" + qs;
+  }
+
+  var options = {
+    loginURL: config.appURL + "/" + locale + "/login",
+    editorHOST: config.editorHOST,
+    editorURL: config.editorURL,
```

Comment:
I don't think we need any of these 3 things in here. Get Gideon to confirm or correct me.